controller/clickhouse.go

```diff
@@ -883,9 +883,19 @@ func (ck *ClickHouseController) GetOpenSessions(c *gin.Context) {
 		return
 	}
 
+	var gotError bool
 	conf = con.(model.CKManClickHouseConfig)
 	sessions, err := clickhouse.GetCkOpenSessions(&conf, limit)
 	if err != nil {
+		gotError = true
+		if exception, ok := err.(*client.Exception); ok {
```

Comment:
errors.As(), refers to https://github.com/housepower/clickhouse_sinker/blob/1208a4b4e6a7a1aa817f54ba1f90036233a7cf90/output/clickhouse.go#L125